### DISCUSSION TITLE
Taxonomy Committee meeting frequency?

### DISCUSSION THREAD

**Discussion by genevieve-anderegg** [ArctosDB/arctos#8108] [General]
Hello @ArctosDB/taxonomy !
I am now leading the Taxonomy committee, and I was wondering if the committee is best meeting every other month, as we do now, or if anyone preferred to meet more frequently. We had our September meeting today with just me, @sharpphyl and @dustymc , and concluded that many of the current open taxonomy issues need author or larger committee attendance in order to properly discuss. 
Because of the lower attendance this month (likely due to my late meeting reminder), does anyone want to have a meeting sooner than the next scheduled date of November 14th? 
[Arctos Taxonomy Project](https://github.com/orgs/ArctosDB/projects/27/views/1)
[Taxonomy Committee Rolling Agenda](https://docs.google.com/document/d/1gxMD9uzsyGl2zKaMitQ_6tH33G9RUoVpwDWzuB2Xxmw/edit?usp=sharing)
Also, please let me know if there are any changes you want to see in this committee.
Thanks! ~Genna

**Comment by camwebb** [comment]
Sorry I missed the meeting - I was on vacation, deep offline. Big items to discuss (esp [#7962](https://github.com/ArctosDB/arctos/issues/7962) and [#7478](https://github.com/ArctosDB/arctos/issues/7478) one of which I'm an author on) - I'll make sure to be there next time. I'd be up for meeting sooner than a month.

**Reply by genevieve-anderegg** [reply]
Hi Cam! I hope you enjoyed your vacation, no worries. Would you like to talk about your issues sooner than our next month in November? If we held an October meeting it would be next Thursday the 10th. If other folks (@ArctosDB/taxonomy ) are interested then we can put something on the calendar; however I won't be able to join myself as I am starting a new intern that day. Would anyone be interested in a meeting and taking notes in my absence?

**Reply by camwebb** [reply]
Thanks, but there's no great hurry. It'll be good if you are there, so let's wait until Nov. 10th.

**Comment by genevieve-anderegg** [comment]
Taxonomy meeting today folks! 10am Pacific / 11am Mountain. The next time we meet is in January, so please come with all your discussion items! @ArctosDB/taxonomy @camwebb @cjconroy 
[Arctos Taxonomy Project](https://github.com/orgs/ArctosDB/projects/27/views/1)
[Taxonomy Committee Rolling Agenda](https://docs.google.com/document/d/1gxMD9uzsyGl2zKaMitQ_6tH33G9RUoVpwDWzuB2Xxmw/edit?usp=sharing)
